sdk/eventhub/Azure.Messaging.EventHubs/CONTRIBUTING.md

```diff
@@ -1,19 +1,70 @@
 ﻿# Contributing 
 
-Coming Soon...
+Thank you for your interest in contributing to the Event Hubs client library.  As an open source effort, we're excited to welcome feedback and contributions from the community.  A great first step in sharing your thoughts and understanding where help is needed would be to take a look at the [open issues](https://github.com/Azure/azure-sdk-for-net/issues?q=is%3Aopen+is%3Aissue+label%3AClient+label%3A%22Event+Hubs%22).
 
-For more details about contributing to the Azure Event Hubs client libraries or our other .NET SDKs, please refer to: [Azure SDK for .NET Contributing Guide](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md)
+ Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use any contribution that you make. For details, visit https://cla.microsoft.com.
```

Comment:
Should the URL to the CLA be linked - or is it intentionally not linked because you do not have 'friendly' text name for me?

Reply:
This is a direct snippet from our legal team, so...  not much room to tweak.